client/src/components/Crypto/Crypto.css

```
@@ -42,4 +42,38 @@ button {
 
 .buttonDiv {
     height: 30px;
+}
+
+#textarea {
```

```suggestion
#textarea, .replyArea {
```
If the styling for textarea and replyArea are the same could you combine them both as I have suggested above?